pkg/client/clientset/versioned/typed/ceph.rook.io/v1/cephnvmeofgateway.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
```

Comment:
nit: 2025 :watch:

Reply:
done

Reply:
When I run `make codegen`, the year is automatically changed to 2018 (`Copyright 2018 The Rook Authors. All rights reserved.`) in `pkg/client/clientset/versioned/typed/ceph.rook.io/v1/cephnvmeofgateway.go`.

Reply:
@travisn @obnoxxx https://github.com/rook/rook/pull/16774